proposals/CSP.md

```diff
@@ -180,22 +180,30 @@ Proposed Changes:
 * This sub-proposal only affects WebAssembly.compileStreaming
   and WebAssembly.instatiateStreaming
   
-### Total Behavior
-
-The desired end state with both of these looks something like this:
-
-Operation | default | unsafe-eval | wasm-unsafe-eval | unsafe-eval + wasm-unsafe-val | w/ SRI ok | w/o SRI bad
---- | --- | --- | --- | -- | -- | --
-WebAssembly.validate | allow | disallow | allow | allow | N/A | N/A
-new WebAssembly.Module | allow | allow if from instantiateCompile | allow | allow | allow if from instantiateCompile | disallow if from instantiateCompile
-WebAssembly.compile | allow | disallow | allow | allow | N/A | N/A
-WebAssembly.compileStreaming | based on SRI | based on SRI | based on SRI | based on SRI | allow | disallow
-WebAssembly.instantiate | allow | allow if from instantiateCompile | allow | allow | allow if from instantiateCompile | disallow if from instantiateCompile
-WebAssembly.instantiateStreaming | based on SRI | based on SRI | based on SRI | based on SRI | allow | disallow
-new WebAssembly.CompileError | allow | allow | allow | allow | allow | allow
-new WebAssembly.LinkError | allow | allow | allow | allow | allow | allow
-new WebAssembly.Table | allow | allow | allow | allow | allow | allow
-new WebAssembly.Memory | allow | allow | allow | allow | allow | allow
+### CSP Policy Application Summary
+
+The checks performed for web assembly operations is summarized as follows:
+
+Operation | default | no unsafe-eval | with wasm-unsafe-eval | with unsafe-eval and wasm-unsafe-val 
```

Comment:
IIUC, `unsafe-eval` implies `wasm-unsafe-eval`.  Given that, could you remove `and wasm-unsafe-eval` from the column title so that this implication is clear?  Otherwise, lgtm, thanks!

Reply:
Fair point. I left the table entries and added a note below the table. Thanks.